src/Models/IoTWorkspaceProject.ts

```diff
@@ -290,30 +287,22 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
         ]
       );
       await functionApp.updateConfigSettings(scaffoldType);
-      await functionApp.load();
       this.componentList.push(functionApp);
     }
   }
 
   /**
-   * Init Azure components according to configs.
-   * Create and load azure components. Push to component list.
+   * load Azure components from configs.
+   * Update component list with azure components.
    * @param scaffoldType scaffold type
    * @param componentConfigs azure component configs
    */
-  private async initAzureComponentsWithConfig(
-    scaffoldType: ScaffoldType,
-    componentConfigs: AzureComponentConfig[]
-  ): Promise<void> {
-    this.validateProjectRootPath("initialize azure components with configurations", scaffoldType);
-
-    const components: { [key: string]: Component } = {};
+  private async loadAzureComponentsWithConfig(componentConfigs: AzureComponentConfig[]): Promise<void> {

```

Comment:
"loadAzureComponentsByConfig" would be a better name?

Reply:
Fixed.